### PR TITLE
Add Contributing.md

### PR DESCRIPTION
**Pull request by dfed**
cc @justinseanmartin @mikelikespie

### CODE REVIEW

File: README_ponyd.rst [hunk 1]
```diff
@@ -99,3 +99,7 @@ Known Issues / Improvements
   ``ponyd update-devtools`` could possibly be updated to patch the incoming
   chrome developer tools to hide these unused tabs.
 
+
+Contributing
+------------
+We’re glad you’re interested in SocketRocket, and we’d love to see where you take it. Please read our `contributing guidelines <https://github.com/square/SocketRocket/blob/master/Contributing.md>`_ prior to submitting a Pull Request.
```
Comment (justinseanmartin) [inline README_ponyd.rst]: s/SocketRocket/PonyDebugger/g

Comment (dfed) [inline README_ponyd.rst]: copy/pasta fails

**Comment by justinseanmartin** [comment]
Otherwise, LGTM

File: README_ponyd.rst [hunk 2]
```diff
@@ -99,3 +99,7 @@ Known Issues / Improvements
   ``ponyd update-devtools`` could possibly be updated to patch the incoming
   chrome developer tools to hide these unused tabs.
 
+
+Contributing
+------------
+We’re glad you’re interested in PonyDebugger, and we’d love to see where you take it. Please read our `contributing guidelines <https://github.com/square/SocketRocket/blob/master/Contributing.md>`_ prior to submitting a Pull Request.
```
Comment (justinseanmartin) [inline README_ponyd.rst]: Link still pointing to the wrong project.

**Comment by justinseanmartin** [comment]
:+1:

**Pull request merged**